app/javascript/components/ShowRating.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+
+const ShowRating = props => {
+  let faIconType;
+  let type = props.type;
+  let value = props.value
+
+  if (type == 'comfort') {
+    faIconType = 'star';
+  } else if (type == 'variance') {
+    faIconType = 'square';
+  }
+
+  let iconEmpty = `fa fa-${faIconType}-o`;
+  let iconFull = `fa fa-${faIconType}`;
+  let classArray = [];
+
+  for (var i = 0; i < value; i++) {
```

Comment:
good! As a fun efficiency challenge, how could you do the three loops below in a single loop?